intel_extension_for_transformers/neural_chat/docker/Dockerfile

```diff
@@ -56,6 +56,11 @@ COPY ${REPO_PATH} /intel-extension-for-transformers
 RUN if [ "$REPO_PATH" == "" ]; then rm -rf intel-extension-for-transformers/* && rm -rf intel-extension-for-transformers/.* ; git clone --single-branch --branch=${ITREX_VER} ${REPO} intel-extension-for-transformers ; fi
 WORKDIR /intel-extension-for-transformers
 
+RUN cd / && git clone https://github.com/intel/neural-speed.git  && \

```

Comment:
pip install neural-speed is okay.
actually neural-speed is already in requirements.txt